tv_apps/smart-system/js/child_window_factory.js

```diff
@@ -132,6 +132,24 @@
     return true;
   };
 
+  ChildWindowFactory.prototype.createPreviewWindow = function(evt) {
```

Comment:
To be consistent with others, I amended the triggering method. It's event handler now.